python/mujinplanningclient/realtimeitl3planningclient.py

```diff
@@ -88,247 +89,518 @@ def __init__(
+            fireandforget: If True, does not wait for the command to finish and returns immediately. The command remains queued on the server. (Default: False)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
         """
         taskparameters = {
             'command': 'ComputeRobotConfigsForCommandVisualization',
             'executiongraph': executiongraph,
-            'commandindex': commandindex,
-        }  # type: Dict[str, Any]
+        }  # type: dict[str, Any]
+        if commandindex != 0:
+            taskparameters['commandindex'] = commandindex
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout, fireandforget=fireandforget)
 
-    def ComputeRobotJointValuesForCommandVisualization(self, program, commandindex=0, timeout=2, fireandforget=False, **kwargs):
-        # type: (Any, Any, float, bool, Any) -> Any
+    def ComputeRobotJointValuesForCommandVisualization(self, program, commandindex=0, timeout=2, fireandforget=False, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
+        # type: (str, int, float, bool, Optional[types.ComputeRobotJointValuesForCommandVisualizationParametersDynamicEnvironmentState], Optional[int], Optional[Any]) -> Optional[Any]
         """
         Args:
             program:
             commandindex: (Default: 0)
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2)
-            fireandforget (bool, optional): If True, does not wait for the command to finish and returns immediately. The command remains queued on the server. (Default: False)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 2)
+            fireandforget: If True, does not wait for the command to finish and returns immediately. The command remains queued on the server. (Default: False)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
         """
         taskparameters = {
             'command': 'ComputeRobotJointValuesForCommandVisualization',
             'program': program,
-            'commandindex': commandindex,
-        }  # type: Dict[str, Any]
+        }  # type: dict[str, Any]
+        if commandindex != 0:
```

Comment:
I wanted to look up if this changes behavior, but I don't even see where this function still exists. Is this deprecated?

Reply:
Might as well minimize the diff, I guess. https://github.com/mujin/mujinplanningclientpy/pull/45/commits/fcfc65c4a78ea6749e91022e19ff2e022a2c3f20